src/main/java/com/jagrosh/giveawaybot/Bot.java

```diff
@@ -123,6 +133,23 @@ public boolean deleteGiveaway(long channelId, long messageId)
         catch(Exception ignore) {}
         return database.giveaways.deleteGiveaway(messageId);
     }
+
+    public RestJDA startRestJDA() {

```

Comment:
This sort of resembles a singleton pattern, should this be turned private or does it have other accesses outside the class?

Reply:
True yes!